cns/restserver/ipam.go

```diff
@@ -478,31 +516,30 @@ func (service *HTTPRestService) MarkExistingIPsAsPendingRelease(pendingIPIDs []s
 	return nil
 }
 
-func (service *HTTPRestService) GetExistingIPConfig(podInfo cns.PodInfo) (cns.PodIpInfo, bool, error) {
-	var (
-		podIpInfo cns.PodIpInfo
-		isExist   bool
-	)
+func (service *HTTPRestService) GetExistingIPConfig(podInfo cns.PodInfo) ([]cns.PodIpInfo, bool, error) {
+	podIPInfo := make([]cns.PodIpInfo, 1)
```

Comment:
we set it to one I think assuming it will always look at one IP. I am going to look into this more before we meet as it may make sense to set it to the number of NCs instead and loop